tests/jointcalTestBase.py

```diff
@@ -74,12 +75,15 @@ def _prep_reference_loader(self, center, radius):
         # Make a copy of the reference catalog for in-memory contiguity.
         self.reference = refLoader.loadSkyCircle(center, radius, filterName='r').refCat.copy()
 
-    def _testJointCalTask(self, nCatalogs, relative_error, absolute_error):
-        """Test parseAndRun for jointcal on nCatalogs, requiring less than some error (arcsec)."""
+    def _testJointCalTask(self, nCatalogs, dist_rms_relative, dist_rms_absolute, pa1):
```

Comment:
Thanks: this one definitely needed a better docstring (and this gave me a reason to search for, and find, the sublimeText AutoDocstring plugin!).